analysed-packages/jquery/version-3.4.1/jquery-3.4.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,4203 @@
+PackageLicenseDeclared: LicenseRef-MIT
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0".903df0)
+ - monk ("4.4.0".903df0)
+ - ojo ("4.4.0".903df0)
+ - scancode ("4.4.0".903df0) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
+
+Relationship: SPDXRef-DOCUMENT DESCRIBES SPDXRef-upload153
+
+
+##--------------------------
+## File Information
+##--------------------------
+
+##File
+
+FileName: jquery-3.4.1.tar.gz/jquery-3.4.1.tar/jquery-3.4.1/external/qunit/qunit.js
+SPDXID: SPDXRef-item491697
+FileChecksum: SHA1: 33c76b37e3944ddbfc9cda0f2f4891094b8f1d97
+FileChecksum: SHA256: 110e6bbfa90f14f29051863e3f81fa7e6fe57bec8544b5406ffff63df06cc1f1
+FileChecksum: MD5: 9791398c7f02fc09351db2070cf509ac
+LicenseConcluded: LicenseRef-Apache-2.0 AND LicenseRef-MIT AND LicenseRef-warranty-disclaimer-3128280e49c8d997f279fe7f6b35d52a
+LicenseComments: <text> The information in the file is:
+
+Licensed under the Apache License, Version 2.0 (the "License");
+
+Thus, we concluded Apache-2.0.
+The information in the file is:
+
+Released under the MIT license
+
+Thus, we concluded MIT. </text>
+LicenseInfoInFile: LicenseRef-Apache-2.0
+LicenseInfoInFile: LicenseRef-MIT
+FileCopyrightText: <text> Copyright jQuery Foundation and other contributors
+Copyright 2006 Google Inc. </text>
+
+##File
+
+FileName: jquery-3.4.1.tar.gz/jquery-3.4.1.tar/jquery-3.4.1/external/sinon/sinon.js
+SPDXID: SPDXRef-item491705
+FileChecksum: SHA1: ed928a35892baf5c628a981da59c0af9a4436127
+FileChecksum: SHA256: 09b0a68a57aeea1cb6e05cb4c83ecbd091642137237eae20c8162f21f85b95bf
+FileChecksum: MD5: c1c3888b83816796abf9283382836c16
+LicenseConcluded: LicenseRef-BSD-3-Clause-47c3c8036e11cf7ea16ef27b11942ff5 AND LicenseRef-MIT AND LicenseRef-Public-domain
+LicenseComments: <text> The information in the file is:
+
+/* Sinon.JS 2.3.7, 2017-07-10, @license BSD-3 */
+
+Thus, we concluded BSD-3-Clause.
+The information in the file is:
+
+Copyright(c) 2013 jake luer <jake@alogicalparadox.com> MIT Licensed
+
+Thus, we concluded MIT.
+The information in the file is:
+
+This is free and unencumbered software released into the public domain.
+
+Thus, we concluded Public domain. </text>
+LicenseInfoInFile: LicenseRef-MIT
+LicenseInfoInFile: LicenseRef-BSD
+LicenseInfoInFile: LicenseRef-Public-domain
+LicenseInfoInFile: LicenseRef-Unlicense
+LicenseInfoInFile: LicenseRef-unknown-license-reference
+LicenseInfoInFile: LicenseRef-bsd-new
+FileCopyrightText: <text> Copyright(c) 2013 jake luer <jake@alogicalparadox.com> MIT Licensed </text>
+
+##File
+
+FileName: jquery-3.4.1.tar.gz/jquery-3.4.1.tar/jquery-3.4.1/external/requirejs/require.js
+SPDXID: SPDXRef-item491702
+FileChecksum: SHA1: 33bc97e78f4905e72b4c1eb2ca0a4662588443e3
+FileChecksum: SHA256: 9485f0917f97fcf4f63a5ea365200ffd57f123f451382a2f9a1ad2e2fd51ac9b
+FileChecksum: MD5: f0cc8bbb2fcef87fc194fecbb632fcfa
+LicenseConcluded: LicenseRef-MIT AND LicenseRef-RequireJS
```

Comment:
Why is there LicenseRef-MIT AND LicenseRef-RequireJS cocnluded as far as I can see there is only the statement 

> Released under MIT license, https://github.com/requirejs/requirejs/blob/master/LICENSE

this means in my opinion this file is licensed under MIT and you can find the license text at the given link - which you have than concluded to RequireJS, which is correct but the MIT conclusion is to much here